spec.html

```diff
@@ -7454,8 +7445,7 @@ <h1>FunctionCreate ( _kind_, _ParameterList_, _Body_, _Scope_, _Strict_ [ , _pro
           1. Set _prototype_ to the intrinsic object %FunctionPrototype%.
         1. If _kind_ is not ~Normal~, let _allocKind_ be `"non-constructor"`.
         1. Else, let _allocKind_ be `"normal"`.
-        1. Let _F_ be FunctionAllocate(_prototype_, _Strict_, _allocKind_).
-        1. Return FunctionInitialize(_F_, _kind_, _ParameterList_, _Body_, _Scope_).
+        1. Return ! OrdinaryFunctionCreate(_prototype_, _ParameterList_, _Body_, _allocKind_, _kind_, _Strict_, _Scope_).
```

Comment:
Is there a reason you didn’t keep the same param order - prototype, strict, allocKind, and then the initialize args?

Reply:
The resulting order didn't make sense to me.

Reply:
At the least, strict seems to me like it should come before body and parameter list, since it determines how it’s parsed.

Reply:
Maybe in the real world, but not in spec-world. In general, the spec can't even ask whether some code is strict until after it's been parsed. And anyhow, parsing is well in the past by the time this operation is called. A function's `[[Strict]]` value is only used to affect the creation of an "arguments" object, and how 'this' gets bound. (I think.)

(Moreover, #1563 eliminates the `_Strict_` parameter, so its position in the parameter list may be a moot point.)